acceptance/openstack/cbr/v3/tags_test.go

```diff
@@ -0,0 +1,60 @@
+package v3
+
+import (
+	"testing"
+
+	"github.com/opentelekomcloud/gophertelekomcloud/acceptance/clients"
+	"github.com/opentelekomcloud/gophertelekomcloud/openstack/cbr/v3/tags"
+	th "github.com/opentelekomcloud/gophertelekomcloud/testhelper"
+)
+
+func TestTags(t *testing.T) {
+	client, err := clients.NewCbrV3Client()
+	th.AssertNoErr(t, err)
+
+	tagsResp := tags.ShowVaultProjectTag(client)
```

Comment:
Please use more descriptive names, here you can use projectTags, for example

Reply:
do .Extract() here

Reply:
done